language-support/ts/daml-types/index.ts

```diff
@@ -79,42 +120,54 @@ export const Unit: Serializable<Unit> = {
 export type Bool = boolean;
 
 /**
- * Companion object of the `Bool` type.
+ * Companion object of the [[Bool]] type.
  */
 export const Bool: Serializable<Bool> = {
   decoder: jtv.boolean,
 }
 
 /**
- * The counterpart of DAML's `Int` type. We represent `Int`s as string in order
- * to avoid a loss of precision.
+ * The counterpart of DAML's `Int` type.
+ *
+ * We represent `Int`s as string in order to avoid a loss of precision.
  */
 export type Int = string;
 
 /**
- * Companion object of the `Int` type.
+ * Companion object of the [[Int]] type.
  */
 export const Int: Serializable<Int> = {
   decoder: jtv.string,
 }
 
 /**
- * The counterpart of DAML's `Numeric` type. We represent `Numeric`s as string
- * in order to avoid a loss of precision. The string must match the regular
- * expression `-?(?:0|[1-9]\d*)(?:\.\d+)?(?:[eE][+-]?\d+)?`.
+ * The counterpart of DAML's `Numeric` type.
+ *
+ * We represent `Numeric`s as string in order to avoid a loss of precision. The string must match
+ * the regular expression `-?(?:0|[1-9]\d*)(?:\.\d+)?(?:[eE][+-]?\d+)?`.
  */
 export type Numeric = string;
+
+/**
+ * The counterpart of DAML's Decimal type.
+ *
+ * In DAML, Decimal's are the same as Numeric with precision 10.
+ *
+ */
 export type Decimal = Numeric;
 
 /**
- * Companion function of the `Numeric` type.
+ * Companion function of the [[Numeric]] type.
  */
 // eslint-disable-next-line @typescript-eslint/no-unused-vars
 export const Numeric = (_: number): Serializable<Numeric> =>
   ({
     decoder: jtv.string,
   })
 
+/**
+ * Companion function of the [[Decimal]] type.
```

Comment:
Is this really a function?

Reply:
a function without an argument ... changed it to object.